src/AppInstallerCLICore/Workflows/InstallFlow.cpp

```diff
@@ -158,6 +158,21 @@ namespace AppInstaller::CLI::Workflow
         }
     }
 
+    void DisplayInstallWarnings(Execution::Context& context)
+    {
+        auto displayInstallWarnings = context.Get<Execution::Data::Installer>().value().DisplayInstallWarnings;
+
+        if (displayInstallWarnings && !context.Args.Contains(Execution::Args::Type::IgnoreWarnings) && !Settings::User().Get<Settings::Setting::InstallIgnoreWarnings>())
+        {
+            bool ignoreWarning = context.Reporter.PromptForBoolResponse(Resource::String::InstallWarning);

```

Comment:
I'm not sure what the use case is here, but I don't like prompting on a warning.  At a minimum I would think that the setting should be ignore/display/prompt with the default being display.

ignore = Don't show them at all
display = Display the warning once the installer is chosen (before download) which gives an interactive user a fair amount of time to react with CTRL+C
prompt = This behavior of prompting after displaying the warnings.

Reply:
I agree that not prompting is better, especially knowing that WinGet is used in many autonomous install/upgrade scenarios